index.bs

```diff
@@ -985,15 +985,29 @@ This is the case of legacy `<script src="…">` elements,
 which can inject into the parser using `document.write(…)`.
 Due to the performance issues above, new features must not do this.
 
-<h3 id="avoid-render-blocking"> Avoid features that block rendering</h3>
+<h3 id="avoid-render-blocking">Avoid features that block rendering</h3>
 
 Features that require resource loading or other operations before rendering the page,
 often result in blank page (or the previous page). The result is a poor user experience.
 
 Consider adding such features only in cases when the overall user experience is improved.
-A canonical example of this is blocking rendering in order to download and process a stylesheet. 
+A canonical example of this is blocking rendering in order to download and process a stylesheet.
 The alternative user experience is a flash of unstyled content, which is undesirable.
 
+<h3 id="naming-of-url-attributes">Name URL-containing attributes based on their primary purpose</h3><!-- w3ctag/design-principles#278 -->
```

Comment:
Nice idea to link back to the issue. I'd make it an actual link that can be navigated.

```suggestion
<h3 id="naming-of-url-attributes">Name URL-containing attributes based on their primary purpose</h3><!-- https://github.com/w3ctag/design-principles/issues/278 -->
```